test/core/end2end/generate_tests.bzl

```diff
@@ -249,11 +259,12 @@ END2END_TESTS = {
     "max_concurrent_streams": _test_options(
         proxyable = False,
         exclude_inproc = True,
+        exclude_minstack = True,
```

Comment:
Is there a case in the max_concurrent_streams test that can run without a deadline?  If so, maybe plumb this into the code instead of just excluding the entire test?

Reply:
we could... but we're probably converging on over-engineering for the amount of testing required right now